### ISSUE TITLE
Spikes are appearing as flipped, or upside-down

### ISSUE THREAD

**Issue by TheAnton205** [SpikeInterface/spikeinterface#2664]
Hello, plotting traces in spike interface seem to plot spikes going down, rather than up. Is it possible to flip the y-axis and make them not upside down?

Below is what I want to see:
![image](https://github.com/SpikeInterface/spikeinterface/assets/33186076/990fd27d-c9cd-4f4d-b741-7d4fdd15b7d6)

Here is what spikeInterface is plotting:
![image](https://github.com/SpikeInterface/spikeinterface/assets/33186076/19cbc7ef-9b49-4455-b05f-ef9890073e7d)

**Comment by zm711** [comment]
Extracellular potentials typically go down? What kind of data are you hoping to plot? Which extractor are you using?

**Comment by TheAnton205** [comment]
We are using openephys for extracting and neuropixels are the type of data we want to plot. The first image is a screenshot from openephys during live recording.

**Comment by zm711** [comment]
I would guess that you have some sort of display issue for openephys. Or a referencing issue. In general extracellular potentials are downward deflecting the way the plot traces are. If you have sample data you can share we could double check for you. But the `get_traces()` function shouldn't flip your data.

Do you expect positive data for any other reason other than that is how the gui is displaying it?

**Comment by zm711** [comment]
Of course. Feel free to open more issues if they come up. Happy to help!
